site.yml

```diff
@@ -20,6 +20,7 @@
   environment: '{{ env }}'
   roles: 
   - { role: base }
+  - { role: dev, run_as: master }
```

Comment:
@erikh I have verified volplugin works with the box created with the new ansible and without these changes. 

If you would like me to test and push this entire PR in a way that works with ansible I am happy to help with that. But I would not push things in a way that we have to revert in next commit the moment next box comes out.